user/models.py

```diff
@@ -1,3 +1,20 @@
+from django.contrib.auth.models import AbstractUser
 from django.db import models
 
-# Create your models here.
+
+class User(AbstractUser):
+    username = models.CharField(max_length=256, blank=False, unique=True)
+    password = models.CharField(max_length=128, blank=False, unique=True)
```

Comment:
unique=True can make some issues in case of hash-collisions or identical passwords. This is severe problem.

blank=False is unnecessary because hach can be created from any data. Especially, if it is salted hash.

Reply:
@ssliadniev  correct  me if it's wrong